Themes/default/ManageSearch.template.php

```diff
@@ -351,45 +341,43 @@ function template_spider_edit()
-			<div class="cat_bar">
-				<h3 class="catbg">', $context['page_title'], '</h3>
-			</div>
-			<div class="information noup">
-				', $txt['add_spider_desc'], '
-			</div>
-			<div class="windowbg noup">
-				<dl class="settings">
-					<dt>
-						<strong><label for="spider_name">', $txt['spider_name'], ':</label></strong><br>
-						<span class="smalltext">', $txt['spider_name_desc'], '</span>
-					</dt>
-					<dd>
-						<input type="text" name="spider_name" id="spider_name" value="', $context['spider']['name'], '">
-					</dd>
-					<dt>
-						<strong><label for="spider_agent">', $txt['spider_agent'], ':</label></strong><br>
-						<span class="smalltext">', $txt['spider_agent_desc'], '</span>
-					</dt>
-					<dd>
-						<input type="text" name="spider_agent" id="spider_agent" value="', $context['spider']['agent'], '">
-					</dd>
-					<dt>
-						<strong><label for="spider_ip">', $txt['spider_ip_info'], ':</label></strong><br>
-						<span class="smalltext">', $txt['spider_ip_info_desc'], '</span>
-					</dt>
-					<dd>
-						<textarea name="spider_ip" id="spider_ip" rows="4" cols="20">', $context['spider']['ip_info'], '</textarea>
-					</dd>
-				</dl>
-				<hr>
-				<input type="submit" name="save" value="', $context['page_title'], '" class="button">
-				<input type="hidden" name="', $context['session_var'], '" value="', $context['session_id'], '">
-				<input type="hidden" name="', $context['admin-ses_token_var'], '" value="', $context['admin-ses_token'], '">
-			</div><!-- .windowbg -->
-		</form>
-	</div><!-- #admincenter -->';
+	<form id="admin_form_wrapper" action="', $scripturl, '?action=admin;area=sengines;sa=editspiders;sid=', $context['spider']['id'], '" method="post" accept-charset="', $context['character_set'], '">
+		<div class="cat_bar">
+			<h3 class="catbg">', $context['page_title'], '</h3>
+		</div>
+		<div class="information noup">
+			', $txt['add_spider_desc'], '
+		</div>
+		<div class="windowbg noup">
+			<dl class="settings">
+				<dt>
+					<strong><label for="spider_name">', $txt['spider_name'], ':</label></strong><br>
+					<span class="smalltext">', $txt['spider_name_desc'], '</span>
+				</dt>
+				<dd>
+					<input type="text" name="spider_name" id="spider_name" value="', $context['spider']['name'], '">
+				</dd>
+				<dt>
+					<strong><label for="spider_agent">', $txt['spider_agent'], ':</label></strong><br>
+					<span class="smalltext">', $txt['spider_agent_desc'], '</span>
+				</dt>
+				<dd>
+					<input type="text" name="spider_agent" id="spider_agent" value="', $context['spider']['agent'], '">
+				</dd>
+				<dt>
+					<strong><label for="spider_ip">', $txt['spider_ip_info'], ':</label></strong><br>
+					<span class="smalltext">', $txt['spider_ip_info_desc'], '</span>
+				</dt>
+				<dd>
+					<textarea name="spider_ip" id="spider_ip" rows="4" cols="20">', $context['spider']['ip_info'], '</textarea>
+				</dd>
+			</dl>
+			<hr>
```

Comment:
SMF is not adding hr before buttons anymore.

Reply:
That's not something I added, but I'll remove them in the next commits.

Reply:
Yea I know :P that's probably something slipped while we are cleaning templates.